.goreleaser.yml

```diff
@@ -0,0 +1,97 @@
+project_name: bosh_exporter
+
+builds:
+  - id: binary
+    dir: .
+    main: ./main.go
+    binary: bosh_exporter
+    env:
+      - CGO_ENABLED=0
+    ldflags:
+      - -s
+      - -w
+      - -X github.com/prometheus/common/version.Version={{.Version}}
+      - -X github.com/prometheus/common/version.Revision={{.FullCommit}}
+      - -X github.com/prometheus/common/version.Branch={{.Branch}}
+      - -X github.com/prometheus/common/version.BuildUser={{.Env.USER}}
+      - -X github.com/prometheus/common/version.BuildDate={{.Date}}
+      - -X code.cloudfoundry.org/cli/version/binaryVersion={{.Version}}
+    goos:
+      - linux
+      - darwin
+      - windows
+    goarch:
+      - amd64
```

Comment:
What about dropping all but amd64 architecture ? I always felt like it was a waste of resources to build and upload that many tarballs
If anybody is impacted by this decision, he can easily open an issue and add his favorite arch in the build matrix

Reply:
It feels to me that it is the default for prometheus exporter so I would stick to it.

Reply:
:ok_hand: